### ISSUE TITLE
Unclosed configuration file causing issues

### ISSUE THREAD

**Issue by cgodine** [ARM-DOE/ACT#374]
The DQO production server has been returning a flood of ResourceWarnings having to do with unclosed resources:

`ResourceWarning: Enable tracemalloc to get the object allocation traceback`

Searching through all of the processes that produce this error, it appears it has to do with a file in the utils area not being closed after being used (act/utils/conf/de421.bsp). Since this file is rather large, I believe it could be a culprit for some performance issues we've been having as well.

**Comment by zssherman** [comment]
Thanks @cgodine for the heads up! I'll take a look and see if I can find a way to fix this.

**Comment by zssherman** [comment]
@cgodine Can you post the full error? The de421.bsp file is being closed in ACT, unless skyfield itself is not closing it properly using their built in .close()

**Comment by zssherman** [comment]
@cgodine Think I figured it out, the kernel for that file doesn't close, despite using .close() i'm going to do a PR and see if a del helps it

**Comment by cgodine** [comment]
@zssherman I put full traceback text in #375.

**Comment by kenkehoe** [comment]
@zssherman  do you want to make a release for us to test on our production system?

**Comment by zssherman** [comment]
@kenkehoe I can do that, I was going to wait on brandon rhodes to see if there was maybe a more robust solution, but this temporary attempted fix might be fine for the mean time.

**Comment by zssherman** [comment]
@kenkehoe Is the ACT on the production systems conda installed or source installed?

**Comment by kenkehoe** [comment]
@zssherman  the ACT on our production system is conda installed.

**Comment by zssherman** [comment]
@kenkehoe Gotcha do we have a way to test this in a similar environment as the production system? I want to make sure the temporary del fix works, if not I want to explore other avenues.

**Comment by kenkehoe** [comment]
Ummmm, maybe? We are having a hard time reproducing the error on our dev system. Let me look into this and see. I assume you have not merged into master yet. Is that true?

**Comment by zssherman** [comment]
I have not, I pinged Brandon as well, waiting to hear back. But if we can test it, if it works I can merge the temp fix and do a release until we find a better solution or the true cause.

**Comment by kenkehoe** [comment]
@cgodine  is this still an issue we need to look into or can we close this out?

**Comment by kenkehoe** [comment]
I think this issue was resolved. I'm going to close. We can reopen if the issue comes back.
